### PR TITLE
add plotting_examples notebook

### PR DESCRIPTION
**Pull request by jackgoldsmith4**

### CODE REVIEW

**Comment by jackgoldsmith4** [comment]
tutorial-like notebook contains examples for functions in the plot module

**Comment by tpoterba** [comment]
+100

File: python/hail/docs/plot.rst [hunk 1]
```diff
@@ -30,3 +30,4 @@ Plot functions in Hail accept data in the form of either Python objects or :clas
 .. autofunction:: scatter
 .. autofunction:: qq
 .. autofunction:: manhattan
+.. autofunction:: manhattan
```
Comment (tpoterba) [inline python/hail/docs/plot.rst]: remove

File: python/hail/docs/tutorials-landing.rst [hunk 2]
```diff
@@ -105,3 +105,14 @@ that are possible (and easy!) in this interface.
 
     9: MatrixTable<tutorials/09-matrixtable.ipynb>
 
+
+Plot Examples
+=============
+
+This notebook contains examples of how to use the plotting functions in Hail's
+plot module.
```
Comment (tpoterba) [inline python/hail/docs/tutorials-landing.rst]: maybe link this?
```
:mod:`.plot` module.
```

**Pull request merged**